code/modules/chemistry/Reagents-Misc.dm

```diff
@@ -244,16 +244,26 @@ datum
 				return
 
 			on_mob_life(var/mob/M, var/mult = 1)
-				if (!M) M = holder.my_atom
-				if (prob(90))
-					M.take_toxin_damage(1 * mult)
-				if (prob(5)) M.emote(pick("twitch", "shake", "tremble","quiver", "twitch_v"))
-				if (prob(8)) boutput(M, "<span style=\"color:blue\">You feel [pick("really buff", "on top of the world","like you're made of steel", "food_energized", "invigorated", "full of energy")]!</span>")
-				if (prob(5))
-					boutput(M, "<span style=\"color:red\">You cannot breathe!</span>")
-					M.setStatus("stunned", max(M.getStatusDuration("stunned"), 20 * mult))
-					M.take_oxygen_deprivation(15 * mult)
-					M.losebreath += (1 * mult)
+				if(holder.get_reagent_amount(src.id) > theraputicamount)
+					if (!M) M = holder.my_atom
+					if (prob(90))
+						M.take_toxin_damage(1 * mult)
+					if (prob(5)) M.emote(pick("twitch", "shake", "tremble","quiver", "twitch_v"))
+					if (prob(8)) boutput(M, "<span style=\"color:blue\">You feel [pick("really buff", "on top of the world","like you're made of steel", "food_energized", "invigorated", "full of energy")]!</span>")
+					if (prob(5))
+						boutput(M, "<span style=\"color:red\">You cannot breathe!</span>")
+						M.setStatus("stunned", max(M.getStatusDuration("stunned"), 20 * mult))
+						M.take_oxygen_deprivation(15 * mult)
+						M.losebreath += (1 * mult)
+				else
+					if (istype(holder) && istype(holder.my_atom) && hascall(holder.my_atom,"remove_stam_mod_regen")) // thought you could cheat the system huh
+						holder.my_atom:remove_stam_mod_regen("aranesp")
+					if (istype(holder) && istype(holder.my_atom) && hascall(holder.my_atom,"remove_stam_mod_max"))
+						holder.my_atom:remove_stam_mod_max("aranesp")
+					if (blood_system && ishuman(M))
+						var/mob/living/carbon/human/H = M
+						H.blood_volume += 1  * mult
+						H.nutrition -= 1  * mult
```

Comment:
This can be brought back a scope. nutrition is a var on humans and shouldn't be dependent on the blood system being enabled.